### ISSUE TITLE
Testsuite to compare the output of 2 doxygen runs

### ISSUE THREAD

**Issue by sloriot** [CGAL/cgal#1801]
In order to keep track of the modification of doxygen, we should be able to compare the output of two runs. WIth `HTML_TIMESTAMP=NO`, it is possible to compare the html page directly but a layout change will result in important diff. Maybe looking at the xml output to see what's get exported is a solution. What is important to check is what classes, functions, members, ... end up in the documention.

**Comment by lrineau** [comment]
I think that, among all the issues assigned to Maxime about Doxygen, this one has the highest priority: the comparison tool will then be used to "check" the results of any modification to our Doxygen documentation processing.

**Comment by maxGimeno** [comment]
I have a python script that outputs text files containing all the namespaces, Enums, classes and members, ... for each package. I also have a bash script that use the first one to compare the differences between two runs. Using this, I tried to compare two versions of the documentation : one from 1_8_13_patched and one from 1_8_4_patched. But all I discovered is that 1_8_13 segfaults if asked to generate xml output.
